### DISCUSSION TITLE
Minor issue on bromite.org

### DISCUSSION THREAD

**Discussion by zhmars** [bromite/bromite#1290] [General]
The discussions button doesn't look well on a 1366x768 screen. And changing [the button width](https://github.com/bromite/bromite.github.io/blob/master/assets/css/style.css#L199) from `179px` to `189px`  fix my issue. But I'm not sure if it is the correct way since the stylesheets are coming from upstream. Maybe we should avoid changing them directly.

![Screenshot 2021-07-21 at 19-02-57 Bromite](https://user-images.githubusercontent.com/18466397/126479961-73964a5d-d66f-4da4-aa9d-bdc11b0a166e.png)

**Comment by csagan5** [comment]
> But I'm not sure if it is the correct way since the stylesheets are coming from upstream. Maybe we should avoid changing them directly.

The proper way should be to not use any `px` at all; for now I have applied your suggestion, thanks.
